doc/theoretical_description_binary_classification.rst

```diff
@@ -6,23 +6,16 @@
 Theoretical Description
 =======================
 
-For binary classification in the distribution-free setting, there are
-different ways of handling uncertainty quantification: calibration,
-probabilistic prediction (consisting in estimating confidence intervals or CIs) 
-and set prediction (consisting in estimating prediction sets or PSs). Gupta et al.
-have established a tripod of theorems that connect these three notions for
-score-based classifier [1]. We will use their notation to present these different concepts.
-
-In MAPIE, we focus specifically on set prediction for multi-class classification
-and on calibration for binary classification (via scikit-learn).
-
-Although set prediction is possible for binary classification, we don't recommend using this setting.
-Instead, we recommend the use of calibration (see more details in the
-Calibration section or by using the
-:class:`~sklearn.calibration.CalibratedClassifierCV` proposed by scikit-learn
-or :class:`~mapie.calibration.MapieCalibrator` proposed in MAPIE
-even if top-Label calibration is actually not really binary use-cases).
-Here is an argument from Gupta et al [1]:
+There are mainly three different ways to handle uncertainty quantification in binary classification:
+calibration (see :doc:`theoretical_description_calibration`), confidence interval (CI) for the probability
+:math:`P(Y \vert \hat{\mu}(X))` and prediction sets (see :doc:`theoretical_description_classification`).
+These 3 notions are tightly related for score-based classifier, as it is shown in [1]. 
+
+Prediction sets can be computed in the same way for multiclass and binary classification with
+:class:`~mapie.calibration.MapieClassifier`, and there are the same theoretical guarantees.
+Nevertheless, prediction sets are often much less informative in the binary case than in the multiclass case [#]_.
+
+.. [#] From Gupta et al [1]:
```

Comment:
I think there is a problem here. Either the footnote is at the bottom of the page (it was the case before, but its number was not displayed, that's why we asked it to be changed), either you include it directly in the main text

Reply:
It was intentional in fact.

Reply:
I will find another way to quote this part.